pyproject_1.toml

```diff
@@ -0,0 +1,11 @@
+[tool.semantic_release]
```

Comment:
also, do you think there is a way to merge both the `pyproject.toml` files, or will there be some conflicts?
because for `uv` specific things, I see a heading of `[tool.uv.workspace]` etc, so can both the files be merged into one?

Reply:
All the dependencies are captured in uv.lock file. Maybe that solves our problem?

Reply:
I think it should, can you do some reading around it? 
hopefully we should be able to merge `pyproject_1.toml` and `pyproject.toml` into a single file

Reply:
do you think we should keep this change as the scope of this PR? happy to take it later as well

Reply:
Yeah I think we should leave it as it is for this PR as a reference of our working project. We can take this up in a separate issue.